docs/config.rst

```diff
@@ -12,6 +12,8 @@ Configuration Options
 ``listeners``
   Comma-separated list of listeners that listen for API requests over either HTTP or HTTPS. If a listener uses HTTPS, the appropriate SSL configuration parameters need to be set as well.
 
+  Schema Registry identities are stored in ZooKeeper and are made up of a hostname and port. If multiple listeners are configured, the first listener's port is used for its identity.
```

Comment:
This config is defined in rest-utils. I thought about adding this line to rest-utils but I thought the generic version would be weird... "Rest App identities...". I suppose I could use polymorphism to remove the generalization. Think I should go that far?